yarn-project/circuits.js/src/contract/artifact_hash.test.ts

```diff
@@ -5,7 +5,7 @@ describe('ArtifactHash', () => {
   it('calculates the artifact hash', () => {
     const artifact = getBenchmarkContractArtifact();
     expect(computeArtifactHash(artifact).toString()).toMatchInlineSnapshot(
-      `"0x0879735688d7e382d8d32503adbc554c755f309d03b430df401ef2c554cc71a5"`,
+      `"0x28d83dfbf4c51a8f79c3f36cfbd8eb36c97b54bb6c42000c0b0bb9b64f600c25"`,
```

Comment:
Why did this change?

Reply:
It has now been updated in master to calculate the hash over an empty contract artifact. I think it was unrelated.